macros/graph_parsing/get_model_relation.sql

```diff
@@ -12,6 +13,10 @@
         )
         %}
 
+        {% if model_ref_node.config.materialized == "ephemeral" %}
```

Comment:
What are the chances we introduce materialization type that could break this again? Or rather should we be checking `if not in ('table', 'view')` instead?

Reply:
@jeremyyeo I'd rather check for the exact type that causes problems because people might have implemented their own custom materializations.